.github/workflows/mypy.yml

```diff
@@ -7,8 +7,12 @@ on:
       - main
   pull_request:
     paths:
-      - "Tools/clinic/**"
       - ".github/workflows/mypy.yml"
+      - "Tools/cases_generator/**"
```

Comment:
FYI, we didn't backport the https://github.com/python/cpython/pull/108090 to the 3.12
case_generator is highly dependent on CPython version (They have quite different design of code codes and execution part) so it it not proper to backport to 3.12

Reply:
Thanks, updated to only include the `Tools/wasm` backport.